src/app/shell/Destiny.m.scss

```diff
@@ -1,5 +1,3 @@
 .content {
   user-select: none;
```

Comment:
This line needs to stay where it is. Otherwise users will no longer be able to select text on the "About" page which is a behavior that currently exists in DIM.